source/multigrid/mg_transfer_matrix_free.cc

```diff
@@ -719,19 +719,21 @@ namespace
                              VectorizedArray<Number> *data)
   {
     Assert(degree > 0, ExcNotImplemented());
-    const unsigned int loop_length = 2*degree+1;
+    const int loop_length = 2*degree+1;
```

Comment:
Unfortunately not. clang reports that I compare signed with unsigned integers in that case.